mycompany-seeq-link-connector-myconnector/src/main/java/com/mycompany/seeq/link/connector/MyConnection.java

```diff
@@ -379,6 +390,40 @@ private void syncCondition(DatasourceSimulator.Alarm alarm) {
         // conditions like this example.
         condition.setMaximumDuration("2h");
 
+        // In order to ensure that we have the correct metadata for the capsule properties. This is a list of
+        // all possible properties for each condition.
+        List<CapsulePropertyInputV1> capsuleProperties = new ArrayList<>();
+
+        // We define each capsule property with a Name and a Unit of Measure
+        CapsulePropertyInputV1 capsuleProperty = new CapsulePropertyInputV1();
```

Comment:
The first capsule property is added separatelly / verbose to match csharp version and have some place for comments